static/js/redux/ui/modals/user_settings_modal.jsx

```diff
@@ -162,6 +162,10 @@ class UserSettingsModal extends React.Component {
         ; */
     const preferences = !this.props.userInfo.FacebookSignedUp ? null : (
       <div>
+        <div className="preference cf">
+          <h3>This is your favorite number: {
+            this.props.userInfo.favorite_num}</h3>
+        </div>
```

Comment:
Looks great! One nit-picky thing: it is React convention to keep the `{this.props.xxxx}` all on one line if it holds a single variable